bin/commander.js

```diff
@@ -754,17 +860,17 @@ async function prepublishPackages() {
 }
 
 program
-	.command( 'prepublish-packages-stable' )
-	.alias( 'npm-stable' )
-	.description( 'Prepublish to npm steps for a stable version of WordPress packages' )
+	.command( 'prepublish-packages-stable-minor' )
+	.alias( 'npm-stable-minor' )
```

Comment:
I'm not sure I understand why this is a "minor"?

Reply:
there needs to be at least `minor` version bump when syncing trunk according to the existing docs. Did I miss something?

Reply:
I think we can leave it as `npm-stable` and add an argument option to the command when we add handling for patch releases.

Reply:
Oh got it 👍 Yeah I think that should be an argument probably

Reply:
Cool, updated with 9791c81.